### ISSUE TITLE
Inconsistent look and feel between prometheus and alertmanager web consoles

### ISSUE THREAD

**Issue by rigtorp** [prometheus/alertmanager#1933]
**What did you do?**
Open prometheus and alertmanager web consoles.

**What did you expect to see?**
Consistent design and theme for both services. They should both have the same look and feel.

**What did you see instead? Under which circumstances?**
The two services have different look and feel. 

**Environment**

* Alertmanager version:

0.17.0

* Prometheus version:

2.10.0

**Comment by simonpasquier** [comment]
TL;DR: the UIs were developed by different people at different times hence different frameworks & look-and-feels.

The topic has been discussed a while ago on the developer mailing list:
https://groups.google.com/d/msg/prometheus-developers/CAUUFwH-8Fg/iMvYCSi6AAAJ

At the time, there was no real agreement on the next steps. FWIW @juliusv has started a rewrite of the Prometheus UI based on React (see https://github.com/prometheus/prometheus/pull/5694).

**Comment by juliusv** [comment]
Yeah, it would be great to make both UIs look consistent (which could be done independent of the underlying JS stack, Elm vs. React vs. just jQuery), but it has never been a high priority. I bet if Prometheus was a company, it would have been a high priority for branding / marketing reasons :) If we want to tackle this then probably it makes sense to get the new Prometheus UI settled first, and then orient the AM UI around that look.

**Comment by stuartnelson3** [comment]
do we have anyone with design chops in the org? or have we talked about hiring someone outside who could help?

**Comment by rigtorp** [comment]
I think the Prometheus design is fine, basic bootstrap admin UI. Nothing
fancy needed. But AM UI looks and behaves differently which is slightly
inconvenient.
